### DISCUSSION TITLE
professional UX review

### DISCUSSION THREAD

**Discussion by ghost** [celzero/rethink-app#1622] [Ideas]
Seeing how many questions there are about the functions, concepts, descriptions, etc. of *RethinkDNS* – also remembering my own questions/confusions from months ago – I think the app really could benefit from a __professional UX review__, maybe even a first-start setup helper.
Considering how __essential__ your teams app is nowadays, I think it can be made more accessible for fresh users, without dumbing it down for advanced/curious users…

Don't get me wrong: it already is well-designed, but hopefully a UX person would help out contributing some hours.

Anyhow, thank you a lot for this great app!

**Comment by ignoramous** [comment]
>  I think the app really could benefit from a professional UX review, maybe even a first-start setup helper.

You'd volunteer to help raise sponsorship for this? I don't know how much such a review would take, but I can ask a few designers I know, if you're game...

**Reply by ghost** [reply]
Well, I'd help out where I can (I'm a graphic designer), but I've never done – for example – help raise sponsorships nor UX/UI (although I've done a little UX problems & UI design for a website…) but overall I certainly haven't the theory & experience to provide that in a professional quality – though, I could try to help anybody out.

I now it's quite rude, but at first I thought the UX person would help out for free or for just a symbolic compensation because it's such an important project.
I was thinking 1 option could be to ask in the _GNOME Design_ (Matrix) chat. I don't think the GNOME design philosophy is all the way compatible with RethinkDNS' goals, but they're professionals, so one might expect them to be flexible when working with a different project.

**Reply by ignoramous** [reply]
> the UX person would help out for free or for just a symbolic compensation because it's such an important project.

There are people like this, but the problem is making sure that their time and effort does not go to waste (which it might if, say, both of us can't agree on the direction for Rethink or their UX style or their philosophy towards designing human-computer interfaces, in general).

> 1 option could be to ask in the GNOME Design (Matrix) chat

I am not active there, but it is worth a shot. Thanks. Rethink would most likely want to follow Material / Material You guidelines.

> Well, I'd help out where I can (I'm a graphic designer)

Nice! Please give us your feedback (either here or as individual github issues or over email, whatever you're comfortable with) and we'll try to implement them.

**Comment by ghost** [comment]
Sorry, for not replying for so long: I had an accident & had to relearn to walk with my leg…
Anyway, here's my suggestion: please look at the whole SVG (there's a lot of pages + onboarding) 
![Onboarding + RethinkDNS dark](https://github.com/user-attachments/assets/f132818e-a39f-40fb-99e0-54841c464514)

My priority was very clear: make it **much easier to quickly setup & use for a (colorblind) new user** (without overwhelming choices & network lingo – although still explained through the info button.)

_Please_ share your thoughts!
I'm very interested, and enjoy constructive feedback.
